docs/csharp/language-reference/statements/selection-statements.md

```diff
@@ -0,0 +1,107 @@
+---
+title: "Selection statements - C# reference"
+description: "Learn about C# selection statements: if and switch."
+ms.date: 08/09/2021
+f1_keywords:
+  - "if_CSharpKeyword"
+  - "else_CSharpKeyword"
+  - "switch_CSharpKeyword"
+  - "case_CSharpKeyword"
+  - "defaultcase_CSharpKeyword"
+helpviewer_keywords:
+  - "if statement [C#]"
+  - "if keyword [C#]"
+  - "else keyword [C#]"
+  - "switch statement [C#]"
+  - "switch keyword [C#]"
+  - "case keyword [C#]"
+  - "default keyword [C#]"
+---
+# Selection statements (C# reference)
+
+The following statements select statements to execute out of a number of possible statements based on the value of an expression:
```

Comment:
typo, I think:

```suggestion
The following statements select statements to execute one of a number of possible statements based on the value of an expression:
```

Reply:
@BillWagner no, not a typo. The `switch` statement executes several statements (all those in one section). `if` statement always selects one, because technically the whole block `{...}` is considered as one embedded statement.

Should it be:
> ...select statements to execute FROM a number of possible statements...

?

Reply:
Thanks for clarifying.  I'll update the suggestion.